### PR TITLE
Documentation was synced to latest changes

### PR DESCRIPTION
**Pull request by TikhomirovSergey**
## Change list

Documentation was synced to latest changes
 
## Types of changes

- [x] No changes in production code.
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Details

I am going to improve WIKI after the merging.

### CODE REVIEW

**Comment by TikhomirovSergey** [comment]
@SrinivasanTarget Could you take a look at this.

File: archive/docs/Installing-the-project.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+```
+<dependency>
+  <groupId>io.appium</groupId>
+  <artifactId>java-client</artifactId>
+  <version>4.1.2</version>
+</dependency>
+```
+
+It currently depends on selenium-java 2.53.1. If it is necessary to use another version of Selenium then you can configure pom.xml as follows:
```
Comment (SrinivasanTarget) [inline archive/docs/Installing-the-project.md]: ```Selenium 3.0```

File: archive/docs/The-starting-of-an-app-using-Appium-node-server-started-programmatically.md [hunk 2]
```diff
@@ -0,0 +1,289 @@
+# Requirements
+- Installed Node.js 0.12 or greater.
```
Comment (SrinivasanTarget) [inline archive/docs/The-starting-of-an-app-using-Appium-node-server-started-programmatically.md]: node>4

File: docs/Functions.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
+Appium java client has some features based on [Java 8 Functional interfaces](https://www.oreilly.com/learning/java-8-functional-interfaces).
```
Comment (SrinivasanTarget) [inline docs/Functions.md]: 👍

File: docs/Touch-actions.md [hunk 4]
```diff
@@ -1 +1,44 @@
-... under constraction
\ No newline at end of file
+Appium server side provides abilities to emulate touch actions. It is possible construct single, complex and multiple touch actions.
```
Comment (SrinivasanTarget) [inline docs/Touch-actions.md]: 👍

**Comment by vikramvi** [comment]
I will also review this next week and get back; if I can contribute as well.

File: archive/docs/The-starting-of-an-app-using-Appium-node-server-started-programmatically.md (hunk 2 above)
Comment (TikhomirovSergey) [inline archive/docs/The-starting-of-an-app-using-Appium-node-server-started-programmatically.md]: Good catch!

File: archive/docs/Installing-the-project.md (hunk 1 above)
Comment (TikhomirovSergey) [inline archive/docs/Installing-the-project.md]: @SrinivasanTarget I think it is not necessary to write about the certain version which java client depends on. The fact that the client depends on Selenium java is much more important.

Comment (SrinivasanTarget) [inline archive/docs/Installing-the-project.md]: Ok fine

**Pull request merged**